SAML/saml_config.sh

```diff
@@ -37,6 +38,144 @@ function print_usage() {
   exit 1
 }
 
+function get_settings_id() {
+  SETTINGS_ID=`curl $CURL_OPTS \
+    -H "Authorization: Bearer $API_TOKEN" \
+    -X GET \
+    ${SETTINGS_ENDPOINT} | jq '.authenticationSettings | .[] | select(.type=="'"${SSO_KEYWORD}"'") | .id'`
+}
+
+function exit_if_no_settings_id() {
+  if [ -z "$SETTINGS_ID" ] ; then
+    echo "No ${SSO_KEYWORD} settings are set"
+    echo
+    exit 0
+  fi
+}
+
+function get_active_settings_type() {
+  ACTIVE_SETTINGS_TYPE=`curl $CURL_OPTS \
+    -H "Authorization: Bearer $API_TOKEN" \
+    -X GET \
+    $ACTIVE_ENDPOINT | jq '.activeSettings | .type'`
+}
+
+function set_provider_variables() {
+  if [ "$IDP" = "okta" ] ; then
+    SIGNED_ASSERTION="true"
+    VALIDATE_SIGNATURE="true"
+    VERIFY_DESTINATION="true"
+    EMAIL_PARAM="email"
+  elif [ "$IDP" = "onelogin" ] ; then
+    SIGNED_ASSERTION="false"
+    VALIDATE_SIGNATURE="true"
+    VERIFY_DESTINATION="true"
+    EMAIL_PARAM="User.email"
+  else
+    echo "IDP is unknown/unspecified. Contact Sysdig Support for assistance."
+    exit 1
+  fi
+
+  if [ -z "$METADATA_URL" ] ; then
+    echo "Must specify a metadata URL (provided from IDP-side configuration)"
```

Comment:
Is this always needed? Can't the customer provide the "metadata xml" instead?

Reply:
This was originally set at the script. I didn't wanted to change anything in how the scripts where invoked so DOCS don't have to run for the next onprem. We can have a task after the release to improve and doc this without this rush. What do you think?

Reply:
I'm ok with your change. Thanks